node/src/service.rs

```diff
@@ -20,19 +19,35 @@ use sp_api::ConstructRuntimeApi;
 use sp_consensus::SlotData;
 use sp_keystore::SyncCryptoStorePtr;
 use sp_runtime::traits::BlakeTwo256;
+use sp_runtime::{generic, OpaqueExtrinsic};
 use std::sync::Arc;
 use substrate_prometheus_endpoint::Registry;
 
 pub use sc_executor::NativeExecutor;
 
+pub type Block = generic::Block<Header, OpaqueExtrinsic>;
 
-// Native executor instance.
+// Native Statemint executor instance.
 native_executor_instance!(
-	pub RuntimeExecutor,
+	pub StatemintRuntimeExecutor,
 	statemint_runtime::api::dispatch,
 	statemint_runtime::native_version,
 );
 
+// Native Statemine executor instance.
+native_executor_instance!(
+	pub StatemineRuntimeExecutor,
+	statemine_runtime::api::dispatch,
+	statemine_runtime::native_version,
+);
+
+// Native Westmint executor instance.
+native_executor_instance!(
+	pub WestmintRuntimeExecutor,
+	statemine_runtime::api::dispatch,
```

Comment:
should probably be westmint?

Reply:
I didn't see this comment :see_no_evil:
Will fix it, good catch.